cmake/Utils.cmake

```diff
@@ -0,0 +1,40 @@
+macro(normalize_path in_path)

```

Comment:
It's hard to understand why both macros are needed here.

Reply:
see comment in latest file.